sys/net/gnrc/network_layer/ipv6/nib/nib.c

```diff
@@ -388,6 +402,8 @@ void gnrc_ipv6_nib_change_rtr_adv_iface(gnrc_netif_t *netif, bool enable)
  */
 static void _handle_mtuo(gnrc_netif_t *netif, const icmpv6_hdr_t *icmpv6,
                          const ndp_opt_mtu_t *mtuo);
+static uint32_t _handle_rdnsso(gnrc_netif_t *netif, const icmpv6_hdr_t *icmpv6,
```

Comment:
needs to be guarded with `#if GNRC_IPV6_NIB_CONF_DNS`, got error:

```
/RIOT/sys/net/gnrc/network_layer/ipv6/nib/nib.c:405:17: error: unused function '_handle_rdnsso' [-Werror,-Wunused-function]
static uint32_t _handle_rdnsso(gnrc_netif_t *netif, const icmpv6_hdr_t *icmpv6,
                ^
1 error generated.
```

otherwise

Reply:
Fixed